### PR TITLE
Deny popups

### PR DESCRIPTION
**Pull request by leong96**
This completes the modal functionality for sending emails to the user for various rejections.

admin.html: Linked forms to the back end.
edit_denied_confirm_mail.txt: message altered to include the denied reason.
provider_account_denied_mail.txt: message altered to include the denied reason.
views.py: Re-imported some classes that were removed previously but are still used in views. Implemented code in the deny functions and the admin view to pass the reason to the context of the email.

### CODE REVIEW

File: grasa_event_locator/views.py [hunk 1]
```diff
@@ -504,7 +509,7 @@ def register(request):
 def resetpw(request):
         i = 0
         if request.method == 'POST':
-                if User.objects.filter(username=request.POST['emailAddr']).exists():
+                if UserAccount.objects.filter(username=request.POST['emailAddr']).exists():
```
Comment (justwheel) [inline grasa_event_locator/views.py:512]: Just curious, why did these queries need to change from `User` -> `UserAccount`?

**Review by justwheel — CHANGES_REQUESTED** [review]
Hey @leong96, the code looks good to me although I had one question I left earlier.

I tried testing these changes manually, and I was not able to reject a new user registration or event edits. When I clicked the "Deny" button, I got a similar [stacktrace](http://dpaste.com/176MQTK) for both. It looks like the `denyUser()` method is executed before the front-end prompts the user for a rejection reason.

Rejecting new events worked for me.

Could you please take another look at this one?

File: grasa_event_locator/views.py (hunk 1 above)
Comment (leong96) [reply grasa_event_locator/views.py:512]: Regarding UserAccount, someone changed the import statement to import User as UserAccount. As such there were instances of "User" and "UserAccount" in the code. Any case of the above was switched to UserAccount for consistency (and to get things working again).

Comment (justwheel) [reply grasa_event_locator/views.py:512]: Oh huh, I didn't even notice. Nice catch!

**Review by justwheel — APPROVED** [review]
I tested this and it all works! Sorry if I muddled this while rebasing. Nice work @leong96. :100: 

Merging! :ocean:

**Pull request merged**